### PR TITLE
Update encryption-at-rest.md

### PR DESCRIPTION
**Pull request by jiayang-zheng**
Add Performance reduction of SM4 encryption.

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?
https://github.com/tikv/tikv/issues/13041

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: encryption-at-rest.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ TiKV 当前不从核心转储 (core dumps) 中排除加密密钥和用户数据
 
 TiKV 使用文件的绝对路径来跟踪已加密的数据文件。一旦 TiKV 节点开启了加密功能，用户就不应更改数据文件的路径配置，例如 `storage.data-dir`，`raftstore.raftdb-path`，`rocksdb.wal-dir` 和 `raftdb.wal-dir`。
 
-SM4 加密只在 v6.3.0 及之后版本的 TiKV 上支持。v6.3.0 之前的 TiKV 仅支持 AES 加密。
+SM4 加密只在 v6.3.0 及之后版本的 TiKV 上支持。v6.3.0 之前的 TiKV 仅支持 AES 加密。SM4 加密最大可能对性能造成50%到80%的回退。
```
Comment (BusyJay) [inline encryption-at-rest.md:31]: 吞吐还是延迟？

Comment (jiayang-zheng) [reply encryption-at-rest.md:31]: 计算性能衰退的时候我主要是用sysbench测得的TPS/QPS来计算的,  我更新一下。

File: encryption-at-rest.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ TiKV 当前不从核心转储 (core dumps) 中排除加密密钥和用户数据
 
 TiKV 使用文件的绝对路径来跟踪已加密的数据文件。一旦 TiKV 节点开启了加密功能，用户就不应更改数据文件的路径配置，例如 `storage.data-dir`，`raftstore.raftdb-path`，`rocksdb.wal-dir` 和 `raftdb.wal-dir`。
 
-SM4 加密只在 v6.3.0 及之后版本的 TiKV 上支持。v6.3.0 之前的 TiKV 仅支持 AES 加密。
+SM4 加密只在 v6.3.0 及之后版本的 TiKV 上支持。v6.3.0 之前的 TiKV 仅支持 AES 加密。SM4 加密最大可能对吞吐造成50%到80%的回退。
```
Comment (TomShawn) [inline encryption-at-rest.md:31]:
```suggestion
SM4 加密只在 v6.3.0 及之后版本的 TiKV 上支持。v6.3.0 之前的 TiKV 仅支持 AES 加密。SM4 加密会对吞吐性能造成 50% 到 80% 的回退。
```

**Review by TomShawn — APPROVED** [review]
LGTM
@BusyJay PTAL

File: encryption-at-rest.md (hunk 1 above)
Comment (jiayang-zheng) [reply encryption-at-rest.md]: [Performance Test Report](https://pingcap.feishu.cn/sheets/shtcnxUtd08fHdaFRb9fVtcQPYf?sheet=GXwRyI)

File: encryption-at-rest.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ TiKV 当前不从核心转储 (core dumps) 中排除加密密钥和用户数据
 
 TiKV 使用文件的绝对路径来跟踪已加密的数据文件。一旦 TiKV 节点开启了加密功能，用户就不应更改数据文件的路径配置，例如 `storage.data-dir`，`raftstore.raftdb-path`，`rocksdb.wal-dir` 和 `raftdb.wal-dir`。
 
-SM4 加密只在 v6.3.0 及之后版本的 TiKV 上支持。v6.3.0 之前的 TiKV 仅支持 AES 加密。
+SM4 加密只在 v6.3.0 及之后版本的 TiKV 上支持。v6.3.0 之前的 TiKV 仅支持 AES 加密。SM4 加密会对吞吐性能造成 50% 到 80% 的回退。
```
Comment (BusyJay) [inline encryption-at-rest.md:31]:
```suggestion
SM4 加密只在 v6.3.0 及之后版本的 TiKV 上支持。v6.3.0 之前的 TiKV 仅支持 AES 加密。SM4 加密会对吞吐造成 50% 到 80% 的回退。
```

**Review by BusyJay — APPROVED** [review]

**Pull request merged**